Week-4/Problem-1 Promise.js

```diff
@@ -0,0 +1,16 @@
+const promise = function(number){
```

Comment:
I assume this question asks you to create a polyfill for promises. and not use promise itself